File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/TKIT_ITEM_INVENTORY.ts
export class TKIT_ITEM_INVENTORY {
    public ORG_GROUP_ID: string;
    public ITEM_TYPE: string;
    public ITEM_ID: string;
    public SERIAL_NO: string;
    public LOT_NO: string;
    public ITEM_QTY?: number;
    public STORAGE_LOCATION: string;
    //public SERVICE_DT_TIME?: Date;
    public SERVICE_DT_TIME: any;
    public STATUS?: boolean;
    public USER_FIELD_1: string;
    public USER_FIELD_2: string;
    public AVAILABILITY?: boolean;
    public OWNER: string;
    public OWNER_TYPE: string;
    public CHECKIN_DATE?: Date;
    public UPDATE_DATE?: Date;
    public UPDATE_USER_ID: string;
    public ROW_INDEX: number;
    public Disable: boolean;
    public ASSET_ID: string;
    public ASSET_IDPART1: string;
}

File: RSPAClient/Core/AtPar.POCOEntities/TKIT_ITEM_INVENTORY.cs
using System;
using System.Collections.Generic;

namespace AtPar.POCOEntities
{
    public partial class TKIT_ITEM_INVENTORY
    {
        public string ORG_GROUP_ID { get; set; }
        public string ASSET_ID { get; set; }
        public string ITEM_TYPE { get; set; }
        public string ITEM_ID { get; set; }
        public string SERIAL_NO { get; set; }
        public string LOT_NO { get; set; }
        public Nullable<int> ITEM_QTY { get; set; }
        public string STORAGE_LOCATION { get; set; }
        public Nullable<System.DateTime> SERVICE_DT_TIME { get; set; }
        public Nullable<bool> STATUS { get; set; }
        public string USER_FIELD_1 { get; set; }
        public string USER_FIELD_2 { get; set; }
        public Nullable<bool> AVAILABILITY { get; set; }
        public Nullable<System.DateTime> CHECKIN_DATE { get; set; }
        public Nullable<System.DateTime> UPDATE_DATE { get; set; }
        public string UPDATE_USER_ID { get; set; }

        //Ignore
        public int ROW_INDEX { get; set; }

       // public Nullable<bool> Disable {get;set;}


    }
}
